geopandas/tests/test_transformer.py

```diff
@@ -0,0 +1,16 @@
+from geopandas._transformer import TransformerFromCRS
+from pyproj import Transformer
+from shapely.geometry import Point
+from shapely.ops import transform
+
+
+class TestTransformer:
+    def test_transformer(self):
+        pyproj_tf = Transformer.from_crs("EPSG:4326", "EPSG:3857", always_xy=True)
+        tf = TransformerFromCRS("EPSG:4326", "EPSG:3857", always_xy=True)
```

Comment:
Given that this cached object won't be public (and in practice is already tested through `to_crs`), I don't think it is needed to add explicit tests for it.

Reply:
Thanks, I originally added this to try and keep the CI happy about the coverage tests, but hopefully the simplification as you suggest will alleviate this :+1: